static/app/components/modals/createReleaseIntegrationModal.tsx

```diff
@@ -0,0 +1,99 @@
+import {Fragment} from 'react';
+
+import {addErrorMessage, addSuccessMessage} from 'sentry/actionCreators/indicator';
+import {ModalRenderProps} from 'sentry/actionCreators/modal';
+import FieldFromConfig from 'sentry/components/forms/fieldFromConfig';
+import Form from 'sentry/components/forms/form';
+import {Field} from 'sentry/components/forms/type';
+import {t, tct} from 'sentry/locale';
+import {Organization, Project} from 'sentry/types';
+import useApi from 'sentry/utils/useApi';
+
+export type CreateReleaseIntegrationModalOptions = {
+  onCancel: () => void;
+  onCreateSuccess: (integration) => void;
+  organization: Organization;
+  project: Project;
+};
+type CreateReleaseIntegrationModalProps = CreateReleaseIntegrationModalOptions &
+  ModalRenderProps;
+
+function CreateReleaseIntegrationModal({
+  Body,
+  Header,
+  closeModal,
+  project,
+  organization,
+  onCreateSuccess,
+  onCancel,
+}: CreateReleaseIntegrationModalProps) {
+  const api = useApi();
+  const fields: Field[] = [
+    {
+      name: 'name',
+      type: 'string',
+
+      placeholder: `${project.name} Release Integration`,
+      label: t('Name'),
+      help: <Fragment>{t('Name of new integration.')}</Fragment>,
+      defaultValue: `${project.name} Release Integration`,
+      required: true,
+    },
+  ];
+  return (
+    <Fragment>
+      <Header>
+        <h3>{t('Create a Release Integration')}</h3>
+      </Header>
+      <Body>
+        <Form
+          onCancel={onCancel}
```

Comment:
onCancel should close the modal, right now i don't think its hooked up to anything